ext/ManoptJuMPExt.jl

```diff
@@ -1,15 +1,17 @@
 module ManoptJuMPExt
 
+using Manopt
 using LinearAlgebra
 using JuMP: JuMP
 const MOI = JuMP.MOI
 using ManifoldsBase
 using ManifoldDiff
+using Manifolds
```

Comment:
Why would you need Manifolds in this extension? There is already a `ManoptManifoldsExt`, so mixing that in here is maybe a bit chaotic?

As far as I can see in the code (and hope for) `ManifoldsBase.jl` should be enough – but that is a strong dependency for Manopt anyways.

Reply:
Yes, I thought so too. If ManifoldsBase is enough, that's a good news